mappot_fitacf

```diff
@@ -0,0 +1,1136 @@
+#
+#----------------------------------------
+#   How to run
+#---------------------------------------
+#   1. Get off your chair and move your legs really fast (joking)
+#   2. step one! Make sure rst is installed 
+#   3. Make sure you have all the files in the git repo: https://github.com/SuperDARNCanada/mapping/tree/master
+#       -> mappot_fitacf
+#       -> instal.sh
+#       -> north_extensions south_extensions 
+#       -> rainbow.key 
+#   4. Setup: 
+#           source install.sh
+#       4.a) if you do not want to run this script everytime you open a terminal then copy the contents of the file into your .bashrc file 
+#               make sure you change all `pwd` commands to the directory they are located. 
+#   5. Simple Run:
+#           cd to/fitcon/data/
+#           mappot_fitcon --date 20171114 --hemi north 
+#      This will generate convection maps for the date 20171114 for the nothern hemisphere
+#       The maps can be found in: 
+#           ./20171114/north/<current date>/
+#           5a. For more options
+#               mappot_fitacf --help
+#
+
+
+
+#What do those exit values mean? 
+
+#----------------------------------------
+#       Exit value list
+#----------------------------------------
+# -1 : command error
+# 0  : successful
+# 1  : not enough arguements
+# 2  : error file not found
+# 3  : empty file
+# 4  : segfault
+# 5  : rst function failure
+# 7  : error bad value
+
+#-----------------------------------------
+#   Error Variables
+#-----------------------------------------
+ERRCMD=-1
+SUCCESS=0
+ERRARG=1
+ERRFILENOTFOUND=2
+ERREMPTYFILE=3
+ERRSEG=4
+ERRRST=5
+ERRVALUE=6
+
+#-------------------------------------------
+#   Default values
+#-------------------------------------------
+
+INTEGRATIONTIME=120 # 120 seconds -> 2 minutes, standard integration time
+MINIMUMRANGE=2  
+#FMAX=30000000
+VEMAX=1000000
+DELAY=600 # 600 seconds -> 10 minutes this is the defualt delay time for omni sattalite information
+#KEYFILE="../rainbow.key"
+#------------------------------------------
+#   Functions to make my life easier 
+#------------------------------------------
+
+
+
+
+#----------------------------------------
+#arguement_check:
+#   simple if statement to check 
+#       if there is enough arguements
+# purpose: to reduce code copying 
+#
+# parameters: 
+#       $1: number of arguements
+#       $2: expected number
+#       $3: Usage message
+#--------------------------------------
+
+function arguement_check(){
+    
+    # $# - number of arguements
+    if [ $# -ne 3 ]
+    then 
+        echo "Error: $# passed in, expecting 3" >&2
+        echo "Usage: arguement_check <number of arguements> <expected number> <usage message>" >&2
+        exit $ERRARG;
+    fi
+
+    if [ $1 -ne $2 ]
+    then
+        echo "Error: $1 passed in, expecting $2" >&2
+        echo "Usage: $3" >&2
+        exit $ERRARG;
```

Comment:
include styleguide error function, and just send error messages through that? Also includes date and time in the err() so that's nice.